### PR TITLE
adding toggle button

### PR DESCRIPTION
**Pull request by SamikshaSankar**

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\assets",
+    "\\assets\\css",
```
Comment (smaranjitghose) [inline .vs/VSWorkspaceState.json:5]: Remove these files and gitignore them please~

File: index.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
```
Comment (smaranjitghose) [inline index.html:1]: This change is not required😅

File: assets/js/app.js [hunk 3]
```diff
@@ -220,6 +229,32 @@ $(function () {
     let bodyElement = $(`body`);
     bodyElement.prepend(header);
     bodyElement.append(footer);
+
```
Comment (smaranjitghose) [inline assets/js/app.js:232]: Set the default theme to dark please

File: assets/js/app.js [hunk 4]
```diff
@@ -13,6 +13,15 @@ let header = $(`
 
 </div>
```
Comment (smaranjitghose) [inline assets/js/app.js:15]: Add appropriate comments

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Overall nice work @SamikshaSankar .

@anushbhatia could you please help her out with the positioning and the UI enhancement of the toggle button.


Note: The color of the Research Page and the cards are hugely messed up in the dark mode.

**Comment by smaranjitghose** [comment]
@SamikshaSankar Could you make use of [this](https://codepen.io/jh3y/details/LYNZwGm) for the toggle animation instead of the emojis

**Comment by smaranjitghose** [comment]
Closed due to inactivity

**Comment by SamikshaSankar** [comment]
@smaranjitghose I am working on this issue

**Pull request closed without merging**